### PR TITLE
feat: extract a note's source url

### PR DESCRIPTION
**Pull request by joaosa**
Tried to emulate how the rest of the codebase works, i.e. also added a
flag to disable extracting the source-url

### CODE REVIEW

File: src/utils/templates/apply-functions/apply-sourceurl-template.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { NoteData } from '../../../models/NoteData';
+import * as P from './../placeholders/sourceurl-placeholders';
+import { applyConditionalTemplate } from './apply-conditional-template';
+
+export const applySourceUrlTemplate = (noteData: NoteData, text: string): string => {
+    console.log("XXX",noteData.sourceUrl);
```
Comment (Cito) [inline src/utils/templates/apply-functions/apply-sourceurl-template.ts:6]: This was probably only meant for debugging.

Comment (joaosa) [reply src/utils/templates/apply-functions/apply-sourceurl-template.ts]: Yes indeed 😅 . I removed it!

**Comment by joaosa** [comment]
Any chance to merge this? :)

**Comment by akosbalasko** [comment]
Hi @joaosa! Sure sure, sorry about the delay, I'm just in a great rush on my other projects. 
Thank you for your contribution, I'll check it asap.

**Review by akosbalasko — APPROVED** [review]

**Comment by akosbalasko** [comment]
Thanks again, great job!

**Pull request merged**